### PR TITLE
Fix warnings in nanvm-lib - mostly mechanically via #[allow(dead_code)]

### PR DESCRIPTION
**Pull request by sasha-gil**

### CODE REVIEW

File: nanvm-lib/src/js/bitset.rs [hunk 1]
```diff
@@ -19,7 +19,9 @@ pub const SIMPLE_SPLIT: (BitSubset64, BitSubset64) = SIMPLE.split(0x0002_0000_00
 
 pub const BOOL: BitSubset64<bool> = SIMPLE_SPLIT.0.cast();
 
+#[allow(dead_code)]
```
Comment (sergey-shandar) [inline nanvm-lib/src/js/bitset.rs:22]: Could we do this on file/project level?

Comment (sasha-gil) [reply nanvm-lib/src/js/bitset.rs:22]: Yes as it turned out we can use #![allow(dead_code)] just once at the top of the file. In this PR we need that in one of mod.rs, but that can also be done at crate level (in lib.rs) - see https://stackoverflow.com/questions/25877285/how-do-you-disable-dead-code-warnings-at-the-crate-level-in-rust.

Comment (sergey-shandar) [reply nanvm-lib/src/js/bitset.rs:22]: Let's do this on a crate level. We will remove it before publishing the first version.

**Review by sergey-shandar — APPROVED** [review]

File: nanvm-lib/src/js/bitset.rs (hunk 1 above)
Comment (sasha-gil) [reply nanvm-lib/src/js/bitset.rs]: Done - but, I do not know how to turn on a CI requirement on having the build warning-free now on. Let's do that in a separate PR.

**Pull request merged**